### PR TITLE
Configure Dependabot for Dockerfiles

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Configure dependabot for Dockerfiles, so it will propose updates to the new versions of base images

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18831

### CODE REVIEW

**Review by nickboldt — APPROVED** [review]
Let's merge this and see what happens. If nothing happens after a few days, we could try a shorter interval to get faster feedback than `weekly`. Are crontab options available, to do @hourly? for example?

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  # Enable version updates for Docker
+  - package-ecosystem: "docker"
+    # Look for a `Dockerfile` in the `build` directory
+    directory: "/build"
+    # Check for updates once a week
+    schedule:
+      interval: "weekly"
```
Comment (sleshchenko) [inline .github/dependabot.yml:9]:
```suggestion
      interval: "weekly"

```

Comment (sleshchenko) [reply .github/dependabot.yml:9]: and yeah, maybe let's start with daily and if it will spam us too much, we'll be able to return back to weekly

**Review by sleshchenko — APPROVED** [review]

**Comment by nickboldt** [comment]
merged and c'picked to 0.3.x

**Pull request merged**